docassemble/VirtualCourtSampleInterviews/data/questions/Multiuser Signatures - docx.yml

```diff
@@ -34,6 +35,15 @@ code: |
     else:
       final_screen
 ---
+mandatory: true
+question: |  
+subquestion: | 
+  This sample interview illustrates how to handle two-party signature question flow and output, if your template is in **docx**. (For **pdf template**, see sample interview **Multiuser Signatures - pdf**.)
```

Comment:
If we were to change the file name,  it would be treated as a new file and we'd lose the history, right? But I can definitely add more explanation in the interview itself though.

When you have the no-sign-in version ready, you can add a separate sample interview.

Reply:
Answer to the name change: I'm not sure about the playground, but if you change the filename in github it should remember the history.